calcom/cal.com
# Calendar Weekly Scheduler

## What does this PR do?

The time has come 

Loom: https://www.loom.com/share/4c38c3f0a58f435dbc29d96d6035c15b

Fixes #5590

## Type of change


## How should this be tested?
- Please pull down this PR
- `yarn install`
- `cd apps/storybook`
- `yarn dev`
- Test the first storybook -> Press `D` to open up the controls section - try changing around props. 
- View the documentation side of the storybook to see information 


---

## Todos

- [ ] Handle hover overlapping events 
- [ ] Current time line 
- [ ] Mobile Support
- [ ] Pixel push
